sklearn/linear_model/tests/test_logistic.py

```diff
@@ -875,3 +875,11 @@ def test_warm_start():
                         assert_greater(2.0, cum_diff, msg)
                     else:
                         assert_greater(cum_diff, 2.0, msg)
+
+
+@ignore_warnings
```

Comment:
Is it me or is 'balanced' and 'auto' are treated the same, see: this [line](https://github.com/scikit-learn/scikit-learn/blob/85223b9bccc283b4dfc338bd36f6049328a6c798/sklearn/linear_model/logistic.py#L635) where the check is against 'balanced' and not 'auto'.